### PR TITLE
Document realtime compositions

### PR DESCRIPTION
**Pull request by sttts**
Fixes https://github.com/crossplane/docs/issues/588.

### CODE REVIEW

File: content/master/concepts/pods.md [hunk 1]
```diff
@@ -130,6 +130,49 @@ same settings for Providers and their managed resources. Applying the
 `--max-reconcile-rate` to Crossplane only controls the rate for
 core Crossplane resources. 
 {{< /hint >}}
+
+##### Enable Realtime Compositions
```
Comment (plumbis) [inline content/master/concepts/pods.md:134]:
```suggestion
##### Enable real time Compositions
```

File: content/master/concepts/pods.md [hunk 2]
```diff
@@ -130,6 +130,49 @@ same settings for Providers and their managed resources. Applying the
...
+
+Realtime compositions are an alpha feature. Alpha features aren't enabled by
+default.
+
+With realtime compositions enabled Crossplane will watch every composed resource
+through a Kubernetes watch. This allows Crossplane to receive events from the
+apiserver within milliseconds when a composed resource changes, e.g. because
+a provider sets the `Ready` condition to true.
```
Comment (plumbis) [inline content/master/concepts/pods.md:142]:
```suggestion
With real time compositions enabled Crossplane watches every composed resource
with a Kubernetes watch. Crossplane receives events from the
Kubernetes API server when a composed resource changes. For example, when
a provider sets the `Ready` condition to `true`.

{{<hint "important" >}}
Real time compositions are an alpha feature. Alpha features aren't enabled by
default. 
{{< /hint >}}
```

Moves the warning after the introduction to improve noticeability. 

Grammar fixes to pass vale and simplify the sentences.

File: content/master/concepts/pods.md [hunk 3]
```diff
@@ -130,6 +130,49 @@ same settings for Providers and their managed resources. Applying the
...
+
+With realtime compositions enabled, `--poll-interval` becomes irrelevant for
+the composite controller.
```
Comment (plumbis) [inline content/master/concepts/pods.md:145]:
```suggestion
With real time compositions enabled, Crossplane doesn't use the `--poll-interval`
settings. 
```

Simplify language

File: content/master/concepts/pods.md [hunk 4]
```diff
@@ -130,6 +130,49 @@ same settings for Providers and their managed resources. Applying the
...
+
+Enable realtime compositions support by
```
Comment (plumbis) [inline content/master/concepts/pods.md:147]:
```suggestion
Enable real time compositions support by
```

**Review by plumbis — CHANGES_REQUESTED** [review]

**Comment by sttts** [comment]
Squashed.

**Review by plumbis — APPROVED** [review]

**Pull request merged**